QuickOSM/ui/dialog.py

```diff
@@ -132,6 +134,12 @@ def __init__(self, iface=None, parent=None):
             Panels.QuickQuery: self.action_xml_qq,
             Panels.Query: self.action_xml_q,
         }
+        icon = QIcon(resources_path('icons', 'QuickOSM.svg'))
+        self.reload_action = QAction(icon, tr("Reload the query in a new file"), self.iface)
```

Comment:
All new variables in `self` must be declared in `__init__`. Pycharm is raising a warning no ?

Reply:
We are in the __init__ function

Reply:
ok ! sorry